### PR TITLE
Cenc xmlns

### PR DESCRIPTION
**Pull request by juliancooper**
Default `ContentProtection` elements should now have `cenc` namespace attributes.

### CODE REVIEW

File: mpd/fixtures/live_profile.mpd [hunk 1]
```diff
@@ -2,22 +2,22 @@
 <MPD xmlns="urn:mpeg:dash:schema:mpd:2011" profiles="urn:mpeg:dash:profile:isoff-live:2011" type="static" mediaPresentationDuration="PT6M16S" minBufferTime="PT1.97S">
   <Period>
     <AdaptationSet mimeType="audio/mp4" segmentAlignment="true" startWithSAP="1" lang="en">
-      <ContentProtection schemeIdUri="urn:mpeg:dash:mp4protection:2011" cenc:default_KID="08e36702-8f33-436c-a5dd60ffe5571e60" value="cenc" xmlns:cenc="urn:mpeg:cenc:2013"></ContentProtection>
```
Comment (skidder) [inline mpd/fixtures/live_profile.mpd]: This line is functionally identical to the value in the PR, just rearranged. I suggest keeping the previous value unless the change is truly needed, in the interest of making the PR easier to review.

File: mpd/fixtures/live_profile.mpd [hunk 2]
```diff
@@ -2,22 +2,22 @@
...
       </ContentProtection>
       <SegmentTemplate duration="1968" initialization="$RepresentationID$/audio/en/init.mp4" media="$RepresentationID$/audio/en/seg-$Number$.m4f" startNumber="0" timescale="1000"></SegmentTemplate>
       <Representation audioSamplingRate="44100" bandwidth="67095" codecs="mp4a.40.2" id="800"></Representation>
     </AdaptationSet>
     <AdaptationSet mimeType="video/mp4" scanType="progressive" segmentAlignment="true" startWithSAP="1">
-      <ContentProtection schemeIdUri="urn:mpeg:dash:mp4protection:2011" cenc:default_KID="08e36702-8f33-436c-a5dd60ffe5571e60" value="cenc" xmlns:cenc="urn:mpeg:cenc:2013"></ContentProtection>
```
Comment (skidder) [inline mpd/fixtures/live_profile.mpd]: Same here, just rearranged.

File: mpd/fixtures/live_profile.mpd (hunk 1 above)
Comment (juliancooper) [inline mpd/fixtures/live_profile.mpd]: Our tests don't check XML equivalency just string equality, so the order of the attributes matter. I'll add an issue.

**Comment by skidder** [comment]
lgtm

**Pull request merged**